EIPS/eip-1.md

```diff
@@ -447,7 +558,7 @@ For each new EIP that comes in, an editor does the following:
 - The title should accurately describe the content.
 - Check the EIP for language (spelling, grammar, sentence structure, etc.), markup (GitHub flavored Markdown), code style
 
-If the EIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+If the EIP isn't ready, the editor will send it back to the author for revision, with specific instructions and possibly a request for review by Technical Peers. 
```

Comment:
```suggestion
If the EIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
```
This is already encompassed by "with specific instructions."

Reply:
Well, formally it is, but a reminder seems in order.  I'll reconsider the wording.